util/get-jwt-claim-array-value-from-string.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * AWS returns array claims on a JWT in a strange way. If you have an array
+ * like this
+ *
+ * ```json
+ * ["864128384660228893", "698902573331070747", "240098361292620967"]
+ * ```
+ *
+ * You'll get a string that looks like this
+ * ```js
+ * "[864128384660228893 698902573331070747 240098361292620967]"
+ * ```
+ *
+ * with no commas or quotes around the items. This is a simple way to parse
+ * out the values and turn them back into an array.
+ *
+ * It will always be an array of strings, even though the input could
+ * theoretically be anything
+ * @param value string
+ */
+export function getJwtClaimArrayValueFromString(value: string): string[] {
```

Comment:
Should this be a more generic function instead? Perhaps a `parseJWTClaims()` or `parseJWT()` that parses the entire thing, and applies this correction where needed.

Reply:
Maybe - the challenge with that sort of approach, and others similar, is how do you know a value is an array? We are assuming that it is an array based on the presence of a [ and ], but that's not full proof...